presto-product-tests/src/test/resources/com/facebook/presto/tests/querystats/single_query_info_response.json

```diff
@@ -56,7 +56,9 @@
     "processedInputPositions": 0,
     "outputDataSize": "0B",
     "outputPositions": 0,
-    "physicalWrittenDataSize": "0B",
+    "writtenOutputPositions": 0,
+    "writtenOutputLogicalDataSize": "0B",
+    "writtenOutputPhysicalDataSize": "0B",
```

Comment:
you need to add intermediate datasize here too after you add the field.  This is what's causing the test failure